workers/reporter/src/test/kotlin/reporter/ReporterRunnerTest.kt

```diff
@@ -696,6 +698,52 @@ class ReporterRunnerTest : WordSpec({
 
             slotInput.captured.licenseTextProvider.getLicenseText(licenseId) shouldBe customLicenseText
         }
+
+        "use the configured how-to-fix text provider" {
+            val format = "testHowToFixTextProvider"
+            val howToFixTextProviderFile = "testHowToFixTextProvider.kts"
+            val jobConfig = ReporterJobConfiguration(
+                formats = listOf(format),
+                howToFixTextProviderFile = howToFixTextProviderFile
+            )
+
+            val reporter = reporterMock(format)
+            every { reporter.generateReport(any(), any(), any()) } returns listOf(tempfile())
+
+            mockReportersAll(format to reporter)
+
+            // Mock the HowToFixTextProvider.fromKotlinScript function to return the mocked object.
+            val mockHowToFixTextProvider = HowToFixTextProvider { "A test How-To-Fix text." }
+            mockkObject(HowToFixTextProvider)
```

Comment:
The `afterEach` block needs to be extended to unmock this object again to avoid interference with other tests. Using `unmockAll` in `afterEach` would also be an option.